newfolder/data/insertText.js

```diff
@@ -2,4 +2,8 @@ function insertText(elemID, text)
 {
     var elem = document.getElementById(elemID);
     elem.innerHTML = text;
-}
\ No newline at end of file
+}
+
+var insertText1 = "Ahoj, tato nabidka je jiz fuc.";

```

Comment:
Toto by mohlo vést hromadě globálních proměnný a něco v té hromadě hledat by bylo nepříjemné. Mám další dva úkoly:
1) Proměnná bude objekt, který bude mít několik properties, a každá bude obsahovat jeden text. Například:
   const texts = { 
      ahoj: "Ahoj...",
      nikdy: "Nikdykde...",
      ...
   }
2) Konstanta text nebude globální, bude přístupná pouze v rámci souboru JS. Jak se to dělá? Nastuduj si "closures javascript"